sphinx/writers/html.py

```diff
@@ -439,6 +439,8 @@ def depart_highlightlang(self, node):
         pass
 
     def visit_download_reference(self, node):
+        if not self.builder.download:
+            return
```

Comment:
Current code usually uses `self.contet.append('')` to be simply depart function.